src/muya/lib/eventHandler/clickEvent.js

```diff
@@ -75,7 +76,26 @@ class ClickEvent {
           contentState.tableToolBarClick(type)
         }
       }
-      // Handler image and inline math preview click
+      // Handle table drag bar click
+      if (target.classList.contains('ag-drag-handler')) {
+        event.preventDefault()
+        event.stopPropagation()
+        const rect = target.getBoundingClientRect()
+        const reference = {
+          getBoundingClientRect () {
```

Comment:
You could also use `getBoundingClientRect: () => {}` like elsewhere.

Reply:
Don't you think `getBoundingClientRect() {}` is more simple? and it's new syntax of es6.